src/core/jit/model.rs

```diff
@@ -33,12 +33,27 @@ impl<V> FromIterator<(String, V)> for Variables<V> {
 }
 
 #[derive(Debug, Clone)]
-pub struct Arg {
+pub struct Arg<Input> {
     pub id: ArgId,
     pub name: String,
     pub type_of: crate::core::blueprint::Type,
-    pub value: Option<async_graphql_value::Value>,
-    pub default_value: Option<async_graphql_value::ConstValue>,
+    pub value: Option<Input>,
+    pub default_value: Option<Input>,
```

Comment:
So `default_value` is defined in the schema, coming from blueprint and value is specific to a query, right?

Reply:
right